modules/nextflow/src/main/groovy/nextflow/script/ScriptBinding.groovy

```diff
@@ -204,6 +205,12 @@ class ScriptBinding extends WorkflowBinding {
         super.getVariables().find { entry -> entry.value?.is(value) }?.getKey()
     }
 
+
+    @CompileDynamic
+    Object invokeDynamicMethod(String name, Object args) {
```

Comment:
What's the different `invokeDynamicMethod` vs `invokeMethod` ?

Reply:
upppsss, sorry it was not removed from the previous approach